src/applications/facility-locator/components/ResultsList.jsx

```diff
@@ -37,6 +44,55 @@ class ResultsList extends Component {
     }
   }
 
+  /**
+   * Returns Result items by type
+   * @param isMobile boolean
+   * @param query object
+   * @param results array list
+   * @returns [] list of results
+   */
+  renderResultItems(isMobile, query, results) {
+    return results.map(r => {
+      let item;
+      switch (query.facilityType) {
+        case 'health':
+        case 'cemetery':
+        case 'benefits':
+        case 'vet_center':
+          item = <VaFacilityResult location={r} query={query} key={r.id} />;
+          break;
+        case 'provider':
+          // Support non va urgent care search through ccp option
+          if (query.serviceType === CLINIC_URGENTCARE_SERVICE) {
+            item = <UrgentCareResult provider={r} query={query} key={r.id} />;
+          } else {
+            item = <CCProviderResult provider={r} query={query} key={r.id} />;
+          }
+          break;
+        case 'pharmacy':
+          item = <PharmacyResult provider={r} query={query} key={r.id} />;
+          break;
+        case 'urgent_care':
+          if (r.type === LocationType.CC_PROVIDER) {
+            item = <UrgentCareResult provider={r} query={query} key={r.id} />;
+          } else {
+            item = <VaFacilityResult location={r} query={query} key={r.id} />;
+          }
+          break;
+        default:
+          item = null;
+      }
+      item = isMobile ? (
+        <div key={r.id} className="mobile-search-result">
```

Comment:
Let's check with Leyda, but I don't think this class is needed, and removing it would make the `isMobile` param unnecessary (better for responsive design).

```css
  .mobile-search-result {
    padding: 1em 0;    // doesn't seem to be needed
    border-top: none;  // has no effect
  }
```

Reply:
let's add this kind of changes to the design review